lib/r10k/cli/version.rb

```diff
@@ -12,7 +12,16 @@ def self.command
         summary 'Print the version of r10k'
 
         run do |opts, args, cmd|
-          puts R10K::VERSION
+          puts "r10k #{R10K::VERSION}"
+          if opts[:verbose]
+            puts RUBY_DESCRIPTION
+            cmdpath = caller.last.match(/\A(.*#{$PROGRAM_NAME}):/).captures.first
+            puts "Command path: #{cmdpath}"
+            puts "Interpreter path: #{Gem.ruby}"
+            if RUBY_VERSION >= '1.9.3'
+              puts "Default encoding: #{Encoding.default_external.name}"
```

Comment:
Do we need to explicitly call `default_external.name`? Will `default_external` work?